src/main/io/osd.c

```diff
@@ -678,13 +678,12 @@ static const char * navigationStateMessage(void)
         case MW_NAV_STATE_LAND_START:
             return OSD_MESSAGE_STR("STARTING EMERGENCY LANDING");
         case MW_NAV_STATE_LAND_IN_PROGRESS:
-            if (!navigationRTHAllowsLanding()) {
-                if (STATE(FIXED_WING)) {
-                    return OSD_MESSAGE_STR("LOITERING AROUND HOME");
-                }
-                return OSD_MESSAGE_STR("HOVERING");
-            }
             return OSD_MESSAGE_STR("LANDING");
+        case MW_NAV_STATE_HOVER_ABOVE_HOME:
+            if (STATE(FIXED_WING))
+                return OSD_MESSAGE_STR("LOITERING AROUND HOME");
+            else
+                return OSD_MESSAGE_STR("HOVERING");
```

Comment:
Could you drop the else and add braces in the if?

Reply:
Drop the else ? Why ?

Reply:
Oh ok :+1:. Done

Reply:
Thanks!